Doc/library/typing.rst

```diff
@@ -428,12 +428,12 @@ value of type :data:`Any` and assign it to any variable::
 
    from typing import Any
 
-   a = None    # type: Any
-   a = []      # OK
-   a = 2       # OK
+   a: Any = None
+   a = []          # OK
+   a = 2           # OK
 
-   s = ''      # type: str
-   s = a       # OK
+   s = ''          # Inferred type of 's' is str
```

Comment:
Why not ‘s: str = “”’? That’s what the original meant and some checkers allow overriding inferred types.

Reply:
That's what I originally had, but @sobolevn convinced me to change it (and after he pointed it out, I decided I agreed with him 🙂) https://github.com/python/cpython/pull/30148#discussion_r770806572